<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QTabWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>515</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>商品出入统计--肖豪</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;进货&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="InputWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="title">
    <string>进货</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,2,7,2" columnstretch="2,3,1,1,2" rowminimumheight="1,2,3,4" columnminimumwidth="1,2,3,4,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>价格</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEditNum">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="CCustomDateEdit" name="dateEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>10</month>
          <day>11</day>
         </datetime>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::YearSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy/M/d</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>日期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>种类</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>数量</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEditPrice">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ExtendedComboBox" name="comboBoxType">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="ConfirmButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>确认进货</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEditRemark">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备注</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="OutputWidget">
   <attribute name="title">
    <string>出货</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,2,7,2" columnstretch="2,2,2,1,1,2">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>流出方向</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>数量</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>日期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>种类</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备注</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="CCustomDateEdit" name="dateEditOutput">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>价格</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEditOutpuPrice">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="lineEditOutputRemark">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ExtendedComboBox" name="comboBoxOutputKind">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="ExtendedComboBox" name="comboBoxOutput">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEditOutputNum">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>确认出货</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="StockWidget">
   <attribute name="title">
    <string>库存</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="ProfitWidget">
   <attribute name="title">
    <string>总利润</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="InfoWidget">
   <attribute name="title">
    <string>说明</string>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CCustomDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">pubui</header>
  </customwidget>
  <customwidget>
   <class>ExtendedComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">pubui</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxType</tabstop>
  <tabstop>lineEditPrice</tabstop>
  <tabstop>lineEditNum</tabstop>
  <tabstop>lineEditRemark</tabstop>
  <tabstop>ConfirmButton</tabstop>
  <tabstop>dateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
